stats.cpp

```diff
@@ -1,5 +1,21 @@
 #include "stats.h"
-
-Stats Statistics::ComputeStatistics(const std::vector<___>& ) {
-    //Implement statistics here
+#include <bits/stdc++.h>
+namespace Statistics {
+    
+    Stats ComputeStatistics(const std::vector<float>& v) {
+        //Implement statistics here
+        Stats obj;
+        auto n = v.size();
```

Comment:
Also take care of naming... v and obj are not self-explanatory

Reply:
Updated the names.